src/app/components/README.md

```diff
@@ -0,0 +1,7 @@
+This repo follows the container component pattern. See ../containers/README.md for an explanation of this.
+
+All components in this repo need to have at least one story and one set of unit tests including the snapshot tests. We expect 90% or greater code coverage of each component.
```

Comment:
Sometimes snapshots tests are not suitable for a component, so we shouldn't prescribe it.